n2y/notion.py

```diff
@@ -320,6 +272,7 @@ def get_database_pages(self, database_id, filter=None, sorts=None):
         notion_pages = self.get_database_notion_pages(database_id, filter, sorts)
         return [self._wrap_notion_page(np) for np in notion_pages]
 
+    @retry_api_call(max_retries)
```

Comment:
Can you remove `max_retries` here and just rely on the default?

Reply:
that is the default, if you checkout main.py to see where this variable is originally created. It needs to be passed here to allow for changes based on the arguments passed to the main function.
```py
13    max_retries = DEFAULT_MAX_RETRIES
...
22    def main(raw_args, access_token):
...
45        global max_retries
46        max_retries = args.max_retries
```

Reply:
@johndgiese

Reply:
Sounds good, thank you!